src/gen/conditions.rs

```diff
@@ -304,6 +304,12 @@ fn parse_args(
                 flags,
             )?))
         }
+        ALWAYS_TRUE => {
+            if (flags & STRICT_ARGS_COUNT) != 0 {
+                check_nil(a, rest(a, c)?)?;
```

Comment:
What does this do?

Reply:
this still requires the condition to have 0 arguments (in mempool mode). I'm still not sure how this always-true condition is meant to be used, so being strict on the argument count is a conservative approach. But perhaps the whole point of this is to allow any arguments, and just always ignore them.

Reply:
in either case, there should be a few test cases covering the arguments

Reply:
Yeah, just ignore any arguments. The idea is to allow for "remarks".